docs/deployment-process/issue-tracking/jira.md

```diff
@@ -76,13 +76,13 @@ This process is for Jira Server, if you are using Jira Cloud, see [Connecting Ji
 
     Ensure the **Is Enabled** property is set.
 
-2. In Octopus Deploy Configure the Release Note Options _(optional)_.
+2. In Octopus Deploy Configure the Release Note Options.
 
-    - **Jira username/password**: Set these values to allow Octopus to connect to Jira and retrieve Jira issue (work item) details when viewing packages or creating releases. If these are not provided, just the raw work item references will be used as the work item link descriptions. If they are provided the work item's title will be used as the work item link's description.
+    - **Jira username/password**: Set these values to allow Octopus to connect to Jira and retrieve Jira issue (work item) details when viewing packages or creating releases. If these are not provided, work items will not be displayed when viewing packages or creating releases. If they are provided the work item's title will be used as the work item link's description.
```

Comment:
I wasn't a fan of the "if not provided then xxx. if provided then yyyy." part. Consider:
```suggestion
    - **Jira username/password**: Set these values to allow Octopus to connect to Jira and retrieve the Jira issue (work item) title when viewing packages or creating releases. Note that if these credentials are not provided, work items will not be displayed when viewing packages or creating releases.
```